autosklearn/data/feature_validator.py

```diff
@@ -34,40 +30,41 @@
 
 class FeatureValidator(BaseEstimator):
     """
-    A class to pre-process features. In this regards, the format of the data is checked,
-    and if applicable, features are encoded
+    Checks the input data to Auto-Sklearn.
+
+    It also determines what columns are categorical and which ones are numerical,
+    so that the pre-processing pipeline can process this columns accordingly.
+
     Attributes
     ----------
-        feat_type: typing.Optional[typing.List[str]]
+        feat_type: typing.Optional[typing.Dict[Union[str, int], str]]
```

Comment:
I think you need to update the description of this attribute.

Reply:
Thanks, I missed this. I have added that.